src/redux/reducers/experimentSettings/setSampleFilterSettingsAuto.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-param-reassign */
+import { produce, original } from 'immer';
+
+const setSampleFilterSettingsAuto = produce((draft, action) => {
+  const { step, sampleId, enabled } = action.payload;
```

Comment:
Maybe you can rename `enabled` to `isAuto`? because `enabled === true` if `auto` is selected right?